docs/common/client_theme.ldoc

```diff
@@ -1,6 +1,15 @@
 */
 
 
+/**
+ * The border color when the client is marked.
+ * It has priority over the rest of beautiful border color properties.
+ * Note that only solid colors are supported.
```

Comment:
The `border_color_`... properties don't support solid colors, not only this one.
Maybe `@param solid_color` should be used instead of adding this note to every one of them?